Meadowlark-js/tests/http/local.ownership-tests.http

```diff
@@ -2,8 +2,8 @@
 # Be sure to load all descriptors into the database first.
 
 
-@auth_header_1=Authorization: bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJlZC1maS1tZWFkb3dsYXJrIiwiYXVkIjoibWVhZG93bGFyayIsInJvbGVzIjpbInZlbmRvciJdLCJzdWIiOiJzdXBlci1ncmVhdC1TSVMiLCJqdGkiOiIyNGMyMDIyMC1mMjg1LTRlNjMtYThmNC1iMmRkYzM5MzAwMDQiLCJpYXQiOjE2NTg1MjM0MDgsImV4cCI6MjY1ODUyNzAwOH0.S2C44HoFMr5ZYSUkbZoAV2pz7Z1Xi40yynH17GOBVoY
-@auth_header_2=Authorization: bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJlZC1maS1tZWFkb3dsYXJrIiwiYXVkIjoibWVhZG93bGFyayIsInJvbGVzIjpbInZlbmRvciJdLCJzdWIiOiJzdXBlci1ncmVhdC1TSVMiLCJqdGkiOiI4MjVkNDRiOC1hODExLTRjZmYtYmVmOC1jYzhhNjYxNjliZTAiLCJpYXQiOjE2NTg1MjQyMzUsImV4cCI6MjY1ODUyNzgzNX0.DR9-ixkAs-EUIsoRjYYGY-NLal2o8wrDYYLcKFfb3_I
+@auth_header_1=Authorization: bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJlZC1maS1tZWFkb3dsYXJrIiwiYXVkIjoibWVhZG93bGFyayIsInN1YiI6InN1cGVyLWdyZWF0LVNJUyIsImp0aSI6IjNkNTliNzVmLWE3NjItNGJhYS05MTE2LTE5YzgyZmRmOGRlMyIsImlhdCI6MTYzNjU2MjA2MCwiZXhwIjozODQ1NTQ4ODgxLCJyb2xlcyI6WyJ2ZW5kb3IiXX0.lUkNIloI8s63UkMK_hlf1iFn0P65hualLPPZcawMQZ4
+@auth_header_2=Authorization: bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJlZC1maS1tZWFkb3dsYXJrIiwiYXVkIjoibWVhZG93bGFyayIsInN1YiI6InNtYWxsLXRvd24tc2lzIiwianRpIjoiMGQwZWZmYTctYWExZi00N2E3LTk4NzktNDdhOGJkOWMzMWJiIiwiaWF0IjoxNjM2NTYyMDc5LCJleHAiOjM4NDU1NDg4ODEsInJvbGVzIjpbInZlbmRvciJdfQ.1nTCwqHcUmuUwNBF8Z2NTHqTg8HPN0mWUQ10TqOCEi0
```

Comment:
Should the third header lower in the file also be updated, so that it uses the new set of credentials?